packages/toolpad-components/src/Text.tsx

```diff
@@ -0,0 +1,120 @@
+import * as React from 'react';
+import {
+  Skeleton,
+  Typography as MuiTypography,
+  TypographyProps as MuiTypographyProps,
+  Link as MuiLink,
+  LinkProps as MuiLinkProps,
+  styled,
+} from '@mui/material';
+import { createComponent, TOOLPAD_COMPONENT_MODE_PROPERTY } from '@mui/toolpad-core';
+import ReactMarkdown from 'react-markdown';
```

Comment:
I wonder whether we should consider lazy loading this

Reply:
Agreed. I imagine it would create a similar layout shift problem as in https://github.com/mui/mui-toolpad/pull/1298#discussion_r1028148876 since the `fallback` element would have a fixed height.

Reply:
The runtime is already wrapped with Suspense wrapper to load components https://github.com/mui/mui-toolpad/blob/6b4a5d6cdcfd733c539980c13dcf9323a05e9508/packages/toolpad-app/src/runtime/ToolpadApp.tsx#L957
So I don't think there would be layout shift, it would just be incorporated in the loading of the components